### ISSUE TITLE
OSC: Fill Screen or die trying

### ISSUE THREAD

**Issue by minimum91** [lhc70000/iina#928]
_I think this is important to make your great application accessible and better received by non-techical users._

Since I've started to use a Mac (with OS X Tiger) for different high-quality h264 media consuption, I've never used any other option but the "Fill Screen" to crop the picture and I don't understand the need for having so many different crop options in a bulbous and intrusive sidebar. **I may be wrong? Possible**, but it seems like overkill for novice or even power users who watch well sourced material (that needs no advanced cropping). Maybe there's room to have both options. One - a simple "Fill screen" button in the OSC, that behaves just like in Quicktime Player, and the other - the currently offered crop features in the sidebar (if those are really needed).

**System and IINA version:**
- macOS 10.11.6
- IINA 0.0.12 Build 37

**Expected behaviour:**
Should have a simple Fill Screen button in the OSC, so that users with well sourced content can easely (with 1 click) adjust the content to the aspect ratio of various displays with different aspect ratios.

**Actual behavior:**
In a simple scenario with a 16:9 movie, I have know how to and to activate the sidebar (the cog icon is always scary for simple users as it often presents hard to understand technical options), then I have to know what screen ratio I have which means either click around to see which fits or looking it up, if you don't know. In a lucky case it's one of the standart offered options, in a worse case, it might be something else, that requires looking it up. In my case it was 25:16. And even then, it turns out, I can't just set a preset to the screen ratio I have.

_**All this hard stuff can be avoided with a simple Fill Screen button.**_

**Comment by saagarjha** [comment]
So, if I'm getting you correctly, you'd like an option in the crop section for your screen's aspect ratio?

**Comment by minimum91** [comment]
Incorrect, that would be a bad idea to have a button for such an unpopular screen aspect ratio. My view is that a 1-click solution [Fill Screen button, just like in Quicktime] for all possible screen aspect ratios is something that IINA would greatly benefit from as it is very accessible, easy to use and understand versus many buttons for each unique screen ratio.

**Comment by saagarjha** [comment]
So a crop option that changes to match the user's current screen ratio?

**Comment by minimum91** [comment]
You could put it that way, yes. That is essentially what the "Fill Screen" button/option in Quicktime does.
